src/OutputFormat.php

```diff
@@ -258,10 +256,7 @@ public function indentWithSpaces(int $numberOfSpaces = 2): self
         return $this->setIndentation(\str_repeat(' ', $numberOfSpaces));
     }
 
-    /**
-     * @return OutputFormat
-     */
-    public function nextLevel()
+    public function nextLevel(): self
```

Comment:
IIUC, we are only PHPDoccing `@return $this fluent interface` when the method returns itself, as opposed to when it returns some relative of the same genus or class.

Reply:
Yes, correct.